src/current/v23.1/operational-faqs.md

```diff
@@ -81,7 +81,7 @@ When MVCC garbage is deleted by garbage collection, the data is still not yet ph
 
 {% include {{page.version.version}}/storage/free-up-disk-space.md %}
 
-## How can I free up disk space quickly?
+## How can I free up disk space that was used by a dropped table?
 
 If you've noticed that [your disk space is not freeing up quickly enough after deleting data](#why-is-my-disk-usage-not-decreasing-after-deleting-data), you can take the following steps to free up disk space more quickly. This example assumes a table `t`.
```

Comment:
Can we also modify this paragraph to mention "dropping a table" instead of "deleting data"?

Reply:
We also may need to note here that if the customer encounters a problem with disk space not being freed up quickly enough after dropping the table, this method will only work for the _next_ table dropped.  If your issue is with the last table dropped, you'll need some other method to identify the ranges that need to be GCed.